src/js/modules/infragistics.ui.tree.js

```diff
@@ -1978,8 +1996,10 @@
 				opt.bindings.childDataProperty = "Nodes";
 				schema.target = { name: "Target", type: "string" };
 				opt.bindings.targetKey = "Target";
-				schema.expanded = { name: "Expanded", type: "boolean" };
-				opt.bindings.expandedKey = "Expanded";
+				schema.expanded = { name: "__expanded__", type: "boolean" };
+				opt.bindings.expandedKey = "__expanded__";
+				schema.checked = { name: "__checked__", type: "boolean" };
```

Comment:
And since I can't add a comment on code outside the change - there's a case with `if (bindings.expandedKey) {` at L2036 that adds `schema.fields.push({ name: "Expanded", type: "boolean" });` to the schema rather than the actual key. And there isn't one for `checkedKey` at all.
Also since there's no XPath flavor for expanded and checked options, should those be working with XML binding at all?